templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
```

Comment:
Do we have to extend basic template?

Reply:
Of course, this is the framework ☺ of our page